examples/dcutr/src/main.rs

```diff
@@ -155,21 +122,30 @@ fn main() -> Result<(), Box<dyn Error>> {
         }
     }
 
-    let behaviour = Behaviour {
-        relay_client: client,
-        ping: ping::Behaviour::new(ping::Config::new()),
-        identify: identify::Behaviour::new(identify::Config::new(
-            "/TODO/0.0.1".to_string(),
-            local_key.public(),
-        )),
-        dcutr: dcutr::Behaviour::new(local_peer_id),
-    };
+    let mut swarm = libp2p::builder::SwarmBuilder::new()
+        .with_existing_identity(generate_ed25519(opts.secret_key_seed))
+        .with_async_std()
+        .with_custom_tcp(tcp::Config::default().port_reuse(true).nodelay(true))
+        .with_relay()
+        .with_other_transport(|keypair| {
+            quic::async_std::Transport::new(quic::Config::new(&keypair))
+                .map(|(peer_id, muxer), _| (peer_id, StreamMuxerBox::new(muxer)))
+        })
+        .no_more_other_transports()
+        .with_dns()
```

Comment:
Similar to another comment, can `with_dns` not impl `no_more_other_transports`?

Reply:
:+1: Done in a6cf98f50.